packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/balance-view/index.css

```diff
@@ -1,70 +1,127 @@
-.card-pay-token-balance-view {
-  display: flex;
-  align-items: start;
-  padding: var(--boxel-sp);
-  box-shadow: 0 0 0 2px var(--boxel-light-400);
+/* details - summary container styles */
+.balance-view {
+  --details-summary-height: 3.125rem; /* 50px */
+  --details-open-height: 11rem; /* ~180px */
+
+  position: relative;
+  border: 1px solid var(--boxel-light-400);
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.35);
   border-radius: var(--boxel-border-radius);
+  height: var(--details-summary-height);
+  overflow: hidden;
+  transition: height var(--boxel-transition);
 }
 
-.card-pay-token-balance-view__currency-details, .card-pay-token-balance-view__balance {
-  flex-shrink: 0;
+.balance-view[open] {
+  height: var(--details-open-height);
+  overflow: auto;
+}
+
+.balance-view__summary {
+  cursor: pointer;
   display: flex;
+  align-items: center;
+  gap: 0 var(--boxel-sp);
+  height: var(--details-summary-height);
+  padding: var(--boxel-sp-xxs) var(--boxel-sp);
+  font: 600 var(--boxel-font-sm);
+  line-height: calc(20/13);
+  transition: padding var(--boxel-transition),
```

Comment:
The padding transition feels like it's lagging behind the rest of the content. I tried tinkering with it for a bit, `transition: font 0.05s;`, skipping padding, feels the most comfortable for me.